src/pages/en/guides/markdown-content.md

```diff
@@ -430,7 +430,9 @@ Use of the `Markdown` component to render remote Markdown can open you up to a [
 
 ## Configuring Markdown
 
-You can customize your Markdown parsing by modifing your `astro.config.mjs`. [Here you can read the full reference](/en/reference/configuration-reference/#markdown-options).
+Markdown support in Astro is powered by [remark](https://remark.js.org/), a powerful parsing and processing tool with an active ecosystem. Other Markdown parsers like Pandoc and markdown-it are not currently supported.
+
+You can customize how remark parses your Markdown in `astro.config.mjs`. See [the reference documentation](/en/reference/configuration-reference/#markdown-options) for full configuration details or follow our guides below on how to add remark plugins and customise syntax highlighting.
```

Comment:
```suggestion
You can customize how remark parses your Markdown in `astro.config.mjs`. See [the reference documentation](/en/reference/configuration-reference/#markdown-options) for full configuration details or follow our guides below on how to add remark plugins and customize syntax highlighting.
```
I think we're typically using North American spellings?  Otherwise, LGTM @delucis !

Reply:
Oops! I even had BOTH spellings in this paragraph…